docs/docs/guides/guides/smart_contracts/writing_contracts/_category_.json

```
@@ -0,0 +1,6 @@
+{
+    "label": "How to Write a Contract",
```

```suggestion
    "label": "Writing Contracts",
```
I think fewer words is better